beacon-chain/interop-cold-start/service.go

```diff
@@ -183,16 +183,20 @@ func (s *Service) saveGenesisState(ctx context.Context, genesisState *stateTrie.
 		return errors.Wrap(err, "could save finalized checkpoint")
 	}
 
+	pubKeys := make([][48]byte, 0)
```

Comment:
can allocate memory here  
```suggestion
	pubKeys := make([][48]byte, 0,genesisState.NumValidators())
```